README.md

```diff
@@ -1,4 +1,15 @@
-[![Travis](https://img.shields.io/travis/rust-lang/rust.svg?style=plastic)](https://github.com/systers/systers.github.io)
-
 # systers.github.io
```

Comment:
@Janiceilene Can we not hyperlink the heading to the site instead?
`# [systers.github.io](http://systers.io/)`